src/styles/blocks/page-section.scss

```diff
@@ -0,0 +1,42 @@
+.page__sectoin {
+  line-height: 33.6px;
+
+  &-title {
```

Comment:
the same problem
if it's modification  '--title' instead '-title'
if it's the element, it would be better 
.page {
  &__section-titile
}